antarest/study/web/output_blueprint.py

```diff
@@ -568,4 +568,37 @@ def get_output_variables_view(
             st_storage_id,
         )
 
+    @bp.post(
+        "/studies/{uuid}/output/{output_id}/variables-views/materialize",
+        summary="Materialize the variables view for a given output and a given configuration",
+    )
+    def materialize_output_variables_view(
```

Comment:
We could add some doc on the get endpoint for the 404 sta5us code meaning and refer to that one